static/js/osmenu.js

```diff
@@ -6,6 +6,7 @@ $(document).ready(function(){
         'el5': [],
```

Comment:
The line above this, should it also add `rhel8`? Haven't looked at the actual code.

Reply:
no, nothing uses "rhel8"